pkg/provisioner/resourcehandlers.go

```diff
@@ -152,73 +158,57 @@ func createConfigMap(obc *v1alpha1.ObjectBucketClaim, ep *v1alpha1.Endpoint, c k
 	return configMap, err
 }
 
+// Only the finalizer needs to be removed. The CM will be deleted since it's
+// ownerReference refers to the parent OBC.
 func deleteConfigMap(cm *corev1.ConfigMap, c kubernetes.Interface) error {
```

Comment:
should probably be renamed since it no longer deletes the CM.  Maybe `releaseConfigMap` or something